tooling/testing.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Testing tx3
+sidebar:
+    order: 5
+---
+
+The testing tool is designed to assist developers in writing tests and validating their tx3 files.
+It provides a structured way to define test cases and verify the expected outcomes of executing tx3 code. 
+
+## Writing
+
+Tests for tx3 files are written using the `TOML` format. A TOML test file typically includes the following sections:
+
+### Wallets
+
+Define the initial state of wallets involved in the test. You can use descriptive names like `bob` and `alice` for clarity. Each wallet should specify its initial balance or assets.
+
+```toml
+[[wallets.bob]]
+lovelace = 10000000
+
+[[wallets.alice]]
+lovelace = 5000000
+assets = { "Token" = 100 }
+```
+
+### Function
+
+Specify the name of the tx3 function that will be executed for this test case.
+
+```toml
+function = "transferTx"
+```
+
+### Inputs
+
+Specify the inputs that the tx3 function under test expects. These inputs should match the parameters defined in your tx3 code.
+
+```toml
+[inputs]
+amount = 1000000
+sender = "bob"
+receiver = "alice"
+```
```

Comment:
no need for this. Args should be specified as part of the transaction as shown in previous comment.